.env

```diff
@@ -3,7 +3,5 @@ KAYRON_HTTP_PORT=7777
 
 KAYRON_LOG_LEVEL=debug
 
-KAYRON_RELEASE_SOURCE=https://github.com/0xSplits/releases
```

Comment:
Removing the default here, because I want the `.env` file to be as generic as possible. Any user specific settings can be applied via env vars as described in the README.